README.md

```diff
@@ -34,12 +34,18 @@ $ bundle exec rails g solidus_related_products:install
 
 You can create four different RelationTypes:
 
-| Applies From | Applies To |
-| - | - |
-| Spree::Product | Spree::Product |
-| Spree::Product | Spree::Variant |
-| Spree::Variant | Spree::Product |
-| Spree::Variant | Spree::Variant |
+| Applies From | Applies To | Bi-Directional available |
+| - | - | - |
+| Spree::Product | Spree::Product | Yes |
+| Spree::Product | Spree::Variant | No  |
+| Spree::Variant | Spree::Product | No  |
+| Spree::Variant | Spree::Variant | Yes |
+
+**Bi-Directional**
+You can optionally set the bi-directional flag (if available) to automatically create the inverse relation.
+Keep in mind that if you remove one side of the relation, also the other hand will be removed, this is not applied to
+the discount and description of the relation, they both continue to work independently and the update on one side
```

Comment:
I think the descriptions and discounts should be synced to be honest, that's what I would expect the gem to do. Also, see my comment above.

Reply:
I've updated the code to keep in sync just the `description`, about the `discount` I've disabled it for the bi-directional relations because I think we can't predict how the consumers of the extension will want this feature to work.